### DISCUSSION TITLE
Error when trying to connect to GitHub

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#235] [Archive] [answered]
Hey,

I keep getting the following  error when trying to connect my package to GitHub,  can someone help?

Error in validate_gh_pat(new_gh_pat(x)) : 
  A GitHub PAT must consist of 40 hexadecimal digits

**Comment by soniamitchell** [comment, marked as answer]
Hey @2199105T, it looks like your PAT isn't set up correctly could you try this? https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat

**Reply by 2199105T** [reply]
That worked! Thank you so  much x
